projects/NodeBB/services/nodebb/examples/postgresql.nix

```diff
@@ -22,8 +22,9 @@
 
   # demo-vm
   networking.firewall.allowedTCPPorts = [ 4567 ];
-  services.getty.helpLine = ''
-    NodeBB needs some time to set up and will list on port 4567 when ready.
-    View journal with `journalctl -efu nodebb.sevice` to see progress.
+  programs.bash.interactiveShellInit = ''
```

Comment:
I think moving this inside the module configuration will be better. We can modify it to only be enabled inside a demo, later on.